src/components/Combobox/Combobox.jsx

```diff
@@ -49,6 +50,7 @@ const Combobox = forwardRef(
       renderOnlyVisibleOptions,
       clearFilterOnSelection,
       maxOptionsWithoutScroll,
+      mountedByKeyboard,
```

Comment:
Combobox  can be auto focused when mounted. in the case of supporting displaying visual focus on first item, we will prefer to show it only if the trigger was keyboard.
We can't know it without prop because the mount trigger happened before our component actually... mount. (Is mounted is ok in terms of grammar and english?)